### ISSUE TITLE
Feature request for HLA analysis

### ISSUE THREAD

**Issue by fazeliniah** [Nesvilab/FragPipe#1034]
Dear MSFragger developing team, 
Thank you again for this amazing software. We are making good progress with using the none-specific HLA workflow in MSFragger.
Our collaborator is interested to do cleavage signature analysis for some the identified peptide. For this purpose, we need the amino acids in the protein sequence that are located at either up or downstream of detected peptides (3 AA either side). Please see the example in Figure 4: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7008090/
I hope you can add this feature and look forward to testing it.
Thanks

**Comment by fcyu** [comment]
Implemented in version 5.0.

**Comment by fazeliniah** [comment]
Hi, I am not sure if I see this new  feature in version 5.0. Maybe I am missing something.

**Comment by ulrich-eckhard** [comment]
Hey Felip! Didn't find time yet to test it actually. Swamped with student assignments but I hope to find time in July to play with these data again! Big thanks already! u

**Comment by ulrich-eckhard** [comment]
> Hi, I am not sure if I see this new feature in version 5.0. Maybe I am missing something.

Hey Fazeliniah!
probably you found it already, but just stumbled about the thread again. i think it is only implemented at the PSM level and not the peptide level. there is now a columns called "Extended Peptide", which has the 8 preceding and the 8 succeeding amino acids (separated from the identified peptide by a dot. so xxxxxxxx.peptide.xxxxxxxx).
cheers,
u

**Comment by ulrich-eckhard** [comment]
Hey MSFragger/FragPipe Team!
Just noticed something odd with the "extended peptide" column: when checking for a specific cleavage site, I realized that the "extended peptide" column didn't have the correct amino acid sequence and instead, two positions that are leucines in the database were indicated as isoleucine. So I checked quickly the amino acid occurrence within the left extended region, and indeed, I only got 0.22% occurrence for leucine, while 12.85% for isoleucine. In the "right" extended peptide region it is not as dramatic but still 2.09 vs 12.54% occurrence, so also likely something going wrong here. Any idea what could be happening?
Big thanks already!
u
